pkg/pod/entrypoint.go

```diff
@@ -141,6 +143,17 @@ func orderContainers(entrypointImage string, commonExtraEntrypointArgs []string,
 			args = append(cmd[1:], args...)
 			cmd = []string{cmd[0]}
 		}
+
+		if breakpointConfig != nil && len(breakpointConfig.Breakpoint) > 0 {
+			breakpoints := breakpointConfig.Breakpoint
+			for _, b := range breakpoints {
+				// TODO: Add other breakpoints
```

Comment:
Can you attach a bug or TEP # here for future readers to followup?

Reply:
Marked the TODO as`TODO(TEP 0042)